src/main/resources/application.yml

```diff
@@ -0,0 +1,24 @@
+server:
+  port: 8080
+  http2.enabled: true
+  compression:
+    enabled: true
+    mime-types: text/html,text/xml,text/plain,text/css,text/javascript,application/javascript,application/json,application/xml
+    min-response-size: 1024
+
+spring:
```

Comment:
Ah, didn't know one could replace the properties with a YAML file.

Reply:
Yes. That's what I suggested since the beginning. YAML drastically reduces redundency and increases readability.

Reply:
@marcauberer True, I remember...